extensions/amp-mathml/0.1/amp-mathml.js

```diff
@@ -33,6 +34,24 @@ export class AmpMathml extends AMP.BaseElement {
     this.preconnect.url('https://cdnjs.cloudflare.com');
   }
 
+  buildCallback() {
+    // Make the element minimally displayed to make sure that `layoutCallback`
+    // is called.
+    let sizingWidth;
+    if (this.element.hasAttribute('inline')) {
+      sizingWidth = '1px';
+    }
+    this.mutateElement(() => {
+      setStyles(this.element, {
+        width: sizingWidth,
+        height: '1rem',
+      });
+    });
+
+    // set data-formula so 3P script case use it.
+    this.element.setAttribute('data-formula',
```

Comment:
This is a bit confusing. Why have the extra `formula` attribute if you're just going to use it to set the `data-formula` attribute and not just use `data-formula`? Basically what happens if the developer sets the `data-formula` attribute? We should at least make this clear in the doc.